api-reference/beta/api/device_post_registeredusers.md

```diff
@@ -32,33 +31,20 @@ Here is an example of the request.
   "name": "create_directoryobject_from_device"
 }-->
 ```http
-POST https://graph.microsoft.com/beta/devices/{id}/registeredUsers
+POST https://graph.microsoft.com/beta/devices/{id}/registeredUsers/$ref
 Content-type: application/json
-Content-length: 30
+Content-length: 97
 
 {
-  "directoryObject": {
-  }
+  "@odata.id":"https://graph.microsoft.com/beta/users/af9f55cc-78ad-4476-be6e-f9bb86a3dddd"
 }
 ```
 In the request body, supply a JSON representation of [directoryObject](../resources/directoryobject.md) object.
```

Comment:
I think this text is not accurate. We are supplying a reference (I think) to the directory object, not a a representation of the directory object. Add group member has somewhat more accurate text for this, but I still think it's a clunky way of saying "reference": https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/group_post_members